src/src/AddBookForm/AddBookForm.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react'
+
+const AddBookForm = () => <div>AddBookForm</div>
+
+export default AddBookForm;
```

Comment:
`ESLint` has a very nice rule of adding a blank line at the end of each file. I love this rule. (But you don't have to use it)

Reply:
On my local branch, where I'm using redux now I have this rule turn on. I will correct the branch we are merging now.